src/Files.App/Views/MainPage.xaml.cs

```diff
@@ -159,9 +159,9 @@ public async void TabItemContent_ContentChanged(object? sender, CustomTabViewIte
 			if (SidebarAdaptiveViewModel.PaneHolder is null)
 				return;
 
-			var paneArgs = e.NavigationParameter as PaneNavigationArguments;
+			PaneNavigationArguments? paneArgs = e.NavigationParameter as PaneNavigationArguments;
```

Comment:
```suggestion
			var paneArgs = e.NavigationParameter as PaneNavigationArguments;
```
There is no need to specify the type explicitly here.